src/ApiKitClient.ts

```diff
@@ -24,61 +30,109 @@ export class ApiKitClient {
     this.setupInterceptors();
   }
 
-  private static setupInterceptors(): void {
+  public static initialize(
+    instanceKey: string,
+    baseURL: string,
+    authTokenCallback?: AuthTokenCallback,
+    unauthorizationCallback?: UnauthorizationCallback,
+    useAuth: boolean = false
```

Comment:
isn't authTokenCallback return false enough instead of having to set 
authTokenCallback + useAuth  for use without auth